features/step_definitions/note_steps.js

```diff
@@ -75,19 +76,52 @@ When(
         );
 
         if (this.response.ok) {
-            const activity = await this.response.clone().json();
-
-            this.activities[noteName] = activity;
-            this.objects[noteName] = activity.object;
+            const responseJson = await this.response.clone().json();
+            const post = responseJson.post;
+            this.posts[noteName] = post;
```

Comment:
I'm not 100% sure about adding a new `this.posts` object :thinking: 

I feel like we could have just used `this.objects` - and either stored the postDTO (which may have been a bit buggy) or map the DTO to an ActivityPub object

That way we can get away with pretty much none of the tests changing 🤔 

We would update the "is in our Outbox" step to have something like

```ts
const activity = this.activities[name];
if (activity) return waitForActivityInOutbox(activity);
const object = this.objects[name];
if (object) return waitFroObjectInOutbox(object);
```

Reply:
This was a great suggestion. Got rid of a lot of test changes by using `this.objects` instead of `this.posts`. Updated tests: https://github.com/TryGhost/ActivityPub/pull/551/commits/ed7cda862e6931a298c4971328ed2316cbc4863e